src/app/components/EditorInterface.tsx

```diff
@@ -170,13 +172,17 @@ const EditorInterface = () => {
 
   useEffect(() => {
     if (appState?.appState.readOnly) return
-    setOriginalTemplate(editor?.getHTML())
+    appState?.setOriginalTemplate(editor?.getHTML() as string)
```

Comment:
we can fallback to empty string

Reply:
2c8a0c529f18e08595783509207e2b1d9209399a I think this tackles the comment @saugatmaharjan